common/helpers/g2p.py

```diff
@@ -0,0 +1,102 @@
+__all__ = ["G2PBasedOovAugmenter"]
+
+import os
+from typing import Optional
+
+from sisyphus import tk
+Path = tk.setup_path(__package__)
+
+from i6_core.corpus.stats import ExtractOovWordsFromCorpusJob
+from i6_core.g2p.apply import ApplyG2PModelJob
+from i6_core.g2p.convert import BlissLexiconToG2PLexiconJob, G2POutputToBlissLexiconJob
+from i6_core.g2p.train import TrainG2PModelJob
+
+
+class G2PBasedOovAugmenter():
+    def __init__(self,
+                 original_bliss_lexicon,
+                 g2p_model_path:Optional[str] = None,
+                 train_args:Optional[dict] = None,
+                 apply_args:Optional[dict] = None,
+                 ):
+        """
+        :param original_bliss_lexicon: path to the original lexicon with OOV
+        :param g2p_model_path: path to the g2p model, if none a g2p model is trained
+        #######################################
+        :param train_args = {
```

Comment:
maybe add the comments about the default params under class directly

Reply:
I would support this, including a short sentence what this class should do.